docs/software/gaming.md

```diff
@@ -349,4 +350,46 @@ Make sure you have one of DietPi's File Servers installed.
 - Steam is installed to the DietPi user data directory `/mnt/dietpi_userdata/steam`
 - By default, installing games will install them to `/mnt/dietpi_userdata/steam`
 
+## PaperMC - Fast and Optimized Minecraft Server, Extendable with Plugins
+
+Descent 1 & 2. A masterpiece 3D FPS. Brought back to life with the DXX-Rebirth project. Play Descent as true to the original with OpenGL/ES rendering.
+
+- DietPi will install the Demo and Shareware versions of Descent. Please see the FAQ below to transfer the full game.
+- Keyboard + Mouse is highly recommended
+- We have compiled the latest version of DXX-Rebirth (0.58.1) with support for FB and RPi OpenGL.
+
+![DietPi gaming software DXX-Rebirth](../assets/images/dietpi-software-games-papermc.jpg){: style="width:500px"}
```

Comment:
Hehe, copy paste from **DDX-Rebirth**? :)

For now we could use the extended title as text: "A highly optimised Minecraft server with plugins, written in Java."
The title itself can be "PaperMC" only, without the short description, as we already decided to apply to all other software titles as well. This also increases the beauty of the scroll link to be simply: `#papermc`

For reference, see the docs in forum which contains some additional short facts: https://dietpi.com/phpbb/viewtopic.php?p=28177#p28177

Reply:
Shall I later (i.e. in a following PR) add the additional information given in  
<https://dietpi.com/phpbb/viewtopic.php?p=28177#p28177>?

Reply:
Would be great! I'm currently working on MineOS, but when this is done I can also do it.